src/plugins/kibana_react/public/page_template/solution_nav/solution_nav.scss

```diff
@@ -0,0 +1,22 @@
+.kbnPageTemplateSolutionNav__title {
+  margin-bottom: $euiSizeL;
+}
+
+@include euiBreakpoint('xs','s') {
+  .kbnPageTemplateSolutionNav {
+    // TODO: Fix in EUI
```

Comment:
Is there an issue or something we can link to here so that others will know when it's safe to remove?

Reply:
Nah, I'll be doing it anyway. It's mostly a reminder to myself.